ci/tasks/build_and_deploy/zip_lambda.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Worth adding a comment to explain the purpose

Reply:
no problem